test/runtests.jl

```diff
@@ -835,3 +835,25 @@ end
     @test x ≈ [2.3, 2.0]
     @test dx ≈ [1.0]
 end
+
+@testset "Batch" begin
+    square(x)=x*x
+    bres = fwddiff(square, BatchDuplicatedNoNeed, BatchDuplicated(3.0, (1.0, 2.0, 3.0)))
+    @test length(bres) == 1
+    @test length(bres[1]) == 3
+    @test bres[1] ≈ (6.0, 12.0, 18.0),
+
+    bres = fwddiff(square, BatchDuplicatedNoNeed, BatchDuplicated(3.0 + 7.0im, (1.0+0im, 2.0+0im, 3.0+0im)))
```

Comment:
I am still against `Duplicated` for immutables. @ZuseZ4 proposed `Dual` the other day as well

Reply:
Sure, but a distincit PR.